### ISSUE TITLE
No sunroof!

### ISSUE THREAD

**Issue by Rcb8004** [Kemmey/Kemmey-TeslaWatch-Public#544]
Replace sunroof and or fart with fold mirrors!  There are no Teslas with a sunroof so why would that be on the list?

**Comment by mobile26** [comment]
Yes, there are many Teslas with sunroofs. So just stay away from changing this function ☀️
